src/App.js

```diff
@@ -63,16 +63,16 @@ export default function App() {
             }}
           />
           <div>
-              <button disabled={!isChrome} onClick={register}>Register</button>
-              <button className="login-button" disabled={!isChrome} onClick={login}>Login</button>
+              <button disabled={!isChrome} onClick={register}>Sign up</button>
```

Comment:
Should we call register here? because our function name is `await magic.webauthn.registerNewUser({ username });` 

Register may be more consistent.

Reply:
Oh just saw this comment - I'm just being consistent with how we're using "Sign up" in the instructions text